trim_peoples_name_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.strip[0..9]
```

Comment:
- [ ] Instead of using direct strip, I recommend 
- ` @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`